examples/informer/README.md

```diff
@@ -0,0 +1,41 @@
+# Informer Example
+
+Informers provide a high-level API for creating custom controllers for Kubernetes resources.
+
+This particular example demonstrates:
+
+* How to write an Informer against a core resource type and handle to create/update/delete events
+
+## Running
+
+``` go
+# assumes you have a working kubeconfig, not required if operating in-cluster 
+go run *.go -kubeconfig=$HOME/.kube/config
+```
+
+## Use Cases
+
+* Capturing resource events for logging to external systems (eg. monitor non-Normal Events and publish metrics to a time series database)
```

Comment:
Folks may want to report events that currently have the Warning type (eg. pod create failures) to external error aggregation services or time-series databases. "Normal" is one of the Types possible on an Event (the other being Warning right now, but the typedef stipulates more may be added later). I'll try and think of some better wording for this.